src/components/Volume/index.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as actions from '../../actions/index';
+import * as toggleTypes from '../../constants/toggleTypes';
+import Slider from 'react-rangeslider';
+
+class VolumeSlider extends React.Component {
```

Comment:
@rwieruch apologies for my delayed response. Thanks for the feedback. I had originally tried to implement this as a stateless component but at the time but didn't get it to work so ended up using the stateful component either though this wasn't the best solution. 
So I am happy to have seen your solution which has improved it. Great! 
Also, good work on the mute button. 👍